card/mod/core/chunk/view_stub.rb

```diff
@@ -4,9 +4,9 @@ module Chunk
       class ViewStub < Abstract
         Chunk.register_class(
           self,
-          prefix_re: Regexp.escape("<card-view>"),
-          full_re: /\<card-view\>([^\<]*)\<\/card-view\>/,
-          idx_char: "<"
+          prefix_re: Regexp.escape("(stub)"),
+          full_re: /\(stub\)([^\(]*)\(\/stub\)/,
+          idx_char: "("
         )
```

Comment:
In case you, like me, have forgotten how this works, each Chunk has a different "idx" character.  Before this was "<", but now it's "(".  Because of the `[^\(]` in the regexp, I think I need to add parenthesis escaping to the stub generator.

Or maybe there's a better regexp...